html/semantics/embedded-content/the-iframe-element/iframe-loading-lazy-nav-form-submit.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<title>Navigating iframe loading='lazy' before it is loaded: form submit</title>
+<iframe name="iframe" src="support/blank.htm?src" loading="lazy"></iframe>
+<form action="support/blank.htm" target="iframe"></form>
+<script>
+const iframe = document.querySelector('iframe');
+const form = document.querySelector('form');
+form.submit();
+</script>
+<!-- Loading testharness.js here is intentional. -->
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script>
+setup({single_test: true});
+onload = () => {
+  step_timeout(() => {
+    assert_equals(iframe.contentWindow.location.href, new URL("support/blank.htm?", location.href).href);
```

Comment:
`support/blank.htm?` matches neither the form navigation (`support/blank.htm`) nor the iframe's default URL `support/blank.htm?src`.

So how did the query param end up there? Is it because of how form navs work? Maybe we could just have `<form action="support/blank.htm?form"` instead.

Reply:
It's because how form submission works, yes. Even with a query in `action`, it still gets populated with the form data (which is empty here). Maybe I can add a comment about this.